scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Not all imports within sections (first party, libraries, our utils) are sorted alphabetically</li><li>Remove unused imports (e.g. from ipywidgets import Video)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>